src/APIFunctions/User.js

```diff
@@ -161,10 +161,10 @@ export async function checkIfUserExists(email) {
  */
 export function filterUsers(users, filterID) {
   let filteredUsers = users.filter((user) => {
-    const d = new Date();
+    // const d = new Date();
     if (filterID === userFilterType.VALID) {
-      return (user.accessLevel >= membershipState.MEMBER
-        && user.membershipValidUntil > d.toISOString());
+      return (user.accessLevel >= membershipState.MEMBER);
+      // && user.membershipValidUntil > d.toISOString());
     } else if (filterID === userFilterType.NON_VALID) {
```

Comment:
This function was already here when I cloned the branch to work on my blast email form. It's supposed to check the validity of the membership for each member in the db, but I think since I'm creating the users (to get emails from each user and send the emails using those) locally, it's messing up the `user.membershipValidUntil` date and is not sending emails properly as a result. If I leave the date stuff in, what I get from fetching the emailList is an empty array.

Reply:
instead of comenting it out just remove it